dynawo/sources/API/CRV/CRVCurve.cpp

```diff
@@ -39,6 +39,16 @@ Curve::const_iterator::const_iterator(const Curve::const_iterator& original) {
 
 Curve::const_iterator::~const_iterator() {
   delete impl_;
+  impl_ = NULL;
+}
+
+Curve::const_iterator&
+Curve::const_iterator::operator=(const Curve::const_iterator& other) {
+  if (this == &other)
+    return *this;
+  delete impl_;
```

Comment:
if ( impl_ != NULL ) delete impl_; ?

Reply:
Not needed there is an embedded test in delete